clustermesh/certs.go

```diff
@@ -137,6 +137,7 @@ func (k *K8sClusterMesh) createClusterMeshClientCertificate(ctx context.Context)
 }
 
 func (k *K8sClusterMesh) createClusterMeshExternalWorkloadCertificate(ctx context.Context) error {
+	certName := getExternalWorkloadCertName()
```

Comment:
Nit. The changes in this file should not be required, since these functions are used only in the classic mode.

Reply:
yeah it's not required, but i'd like to use `getExternalWorkloadCertName()` function throughout the code base consistently. when people see `defaults.ClusterMeshExternalWorkloadSecretName` being used like this, people might get the wrong impression that this is the correct secret name regardless of which mode the CLI is in.